src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -509,8 +510,6 @@ private unsafe void RemoveNodeFromDatabase(TransactionOperationContext context,
                 var updated = EntityToBlittable.ConvertEntityToBlittable(databaseRecord, DocumentConventions.Default, context);
 
                 UpdateValue(index, items, lowerKey, key, updated);
-
-                NotifyDatabaseChanged(context, databaseName, index, nameof(RemoveNodeFromDatabaseCommand));
```

Comment:
Why did you removed this? Shouldn't we notify the db that it has been deleted? How are deletes actually supposed to work now?